### ISSUE TITLE
Scan an ARM64 cluster failed

### ISSUE THREAD

**Issue by nlamirault** [heptio/sonobuoy#520]
**What steps did you take and what happened:**

Perform a scan on an ARM64 Kubernetes cluster
Due to #181 , there is an arm64 version for `sonobuoy-arm64`, but what about others heptio images ? : 

* scanner-forwarder:v0.0.4
* namespace-deleter:v0.0.1

**What did you expect to happen:**

Error of pod : 

```
$ kubectl logs sonobuoy -n heptio-sonobuoy -c forwarder
standard_init_linux.go:190: exec user process caused "exec format error"
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Sonobuoy version: v0.11.6 arm64
- Kubernetes version: (use `kubectl version`): v1.9.6
- Kubernetes installer & version:
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`): HypriotOS ARM64
- Sonobuoy tarball (which contains * below)

**Comment by dims** [comment]
Looks like https://github.com/heptiolabs/namespace-deleter needs to be updated to multi-arch manifests. Can't seem to find a git repo for scanner-forwarder

**Comment by timothysc** [comment]
At this stage it may be a stretch to update scanner for arm, but definitely want to enable sonobuoy proper.

**Comment by lubinsz** [comment]
@nlamirault 
@dims 
@timothysc 

I have enabled sonobuoy to support multi-arch.
I will continue to enable more.
 
At present, there are still 4  in total that do not support multi-arch:
kube-conformance: https://github.com/heptio/kube-conformance 
sonobuoy-plugin-systemd-logs: https://github.com/heptio/sonobuoy-plugin-systemd-logs 
namespace-deleter:  https://github.com/heptiolabs/namespace-deleter 
scanner-forwarder:  ???

If there are new discoveries, please feel free to comment. 
Thanks.

**Comment by dims** [comment]
@lubinsz Do you want to poke at the other end? run e2e tests without using Sonobuoy?

https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md

please see the ppc64le folks who do the same and publishing results here:

https://k8s-testgrid.appspot.com/sig-node-ppc64le#conformance

If you click on any of the green boxes, and navigate down ... you will see the build logs:

https://storage.googleapis.com/ppc64le-kubernetes/logs/ci-conformance-kubernetes/1571/build-log.txt

**Comment by lubinsz** [comment]
@liztio @chuckha 
Hi,
Do you know where is the Dockerfile of 'namespace-deleter:v0.0.1' ?
I need to enable it to support mulit-arch.

Thanks.

**Comment by dims** [comment]
@lubinsz see https://github.com/heptiolabs/namespace-deleter/blob/master/Dockerfile

**Comment by lubinsz** [comment]
@dims 
Sorry, my mistake. 
I need the Dockerfile for 'scanner-forwarder '

**Comment by nlamirault** [comment]
@lubinsz I don't find any source code for scanner-forwarder. Is it open source ?

https://github.com/search?q=scanner-forwarder&type=Code

**Comment by chuckha** [comment]
That project is not open source. We have had many requests to open source it though.

 cc @timothysc

**Comment by johnSchnake** [comment]
At this time nothing has changed with the open sourced nature of scanner-forwarder.

To update this though, of the 4 images listed:

> kube-conformance: https://github.com/heptio/kube-conformance

This is now managed upstream and I think has arm64 arch available. This is only available for k8s clusters >=1.13 and must be specified on the command line as the image. Sonobuoy 1.14 (future release) will default clusters >=1.13 to the upstream image.

> sonobuoy-plugin-systemd-logs: https://github.com/heptio/sonobuoy-plugin-systemd-logs

Not a solution but a comment: You can just disable the systemd plugin via outputting the yaml by `sonobuoy gen`, removing the systemd plugin code, then running it via kubectl.

> namespace-deleter: https://github.com/heptiolabs/namespace-deleter
> scanner-forwarder: ???

Both of these are relevent only to scanner; again I dont know that Scanner will be updated to support arm64 any time soon but if you just use the cmdline Sonobuoy you shouldnt need to worry about these items.

Leaving this opening and marking as scanner specific.

**Comment by johnSchnake** [comment]
Closing; the Scanner is no longer supported at this time. Thank you for your interest in that project. If you have other Sonobuoy questions or issues please feel free to reach out.
